doc/02_Pattern sicurezza/05_accesso-del-fruitore/03_id_auth_rest_01.rst

```diff
@@ -174,7 +174,7 @@ implementative e includono:
 -  algoritmi di firma e digest (alg).
 
 Le parti, in base alle proprie esigenze, individuano gli specifici
-algoritmi secondo quanto indicato al capitolo 7 Elementi di sicurezza
+algoritmi secondo quanto indicato al capitolo 7 "Elementi di sicurezza"
```

Comment:
@ioggstream @mfortini abbiamo un problema, il presente capitolo non è più presente, dobbiamo sostituire con 


Le parti, in base alle proprie esigenze, individuano gli specifici algoritmi secondo quanto indicato nelle Linee Guida sulla sicurezza, emanate dall'Agenzia per l'Italia Digitale ai sensi dell'articolo 71 del decreto legislativo 7 marzo 2005, n. 82 (Codice dell'Amministrazione Digitale).

Reply:
@ioggstream @mfortini lo stesso problema lo abbiamo in:

- doc/02_Pattern sicurezza/04_sicurezza-di-canale/01_id_auth_channel_01.rst
- doc/02_Pattern sicurezza/04_sicurezza-di-canale/02_id_auth_channel_02.rst
- doc/02_Pattern sicurezza/05_accesso-del-fruitore/01_id_auth_soap_01.rst
- doc/02_Pattern sicurezza/05_accesso-del-fruitore/02_id_auth_soap_02.rst
- doc/02_Pattern sicurezza/05_accesso-del-fruitore/04_id_auth_rest_02.rst
- doc/02_Pattern sicurezza/06_integrita/01_integrity_soap.rst
- doc/02_Pattern sicurezza/06_integrita/02_integrity_rest.rst

Reply:
@mfortini fai tu fixing?